modules/dailymotionBidAdapter.md

```diff
@@ -143,6 +146,7 @@ const adUnits = [
           isCreatedForKids: false,
           videoViewsInSession: 1,
           autoplay: false,
+          playerName: 'dailymotion',
```

Comment:
Maybe as an example we should not use dailymotion as `playerName`? Could be confused for a hard-coded value to set

Reply:
Good point, I'll use another player for examples